TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -363,10 +450,16 @@ namespace Plugin {
         CHECK_TTS_MANAGER_RETURN_ON_FAIL();
         _adminLock.Lock();
         TTS::SpeechState state;
-        auto status = _ttsManager->getSpeechState(speechid, state);
+        auto status = TTS::TTS_FAIL;
+
+        if(InputValidation::Instance().validate("speechid", speechid))
+            status = _ttsManager->getSpeechState(speechid, state);
+
         _adminLock.Unlock();
-        if(status == TTS::TTS_OK)
+
+        if(status == TTS::TTS_OK) {
```

Comment:
This addition of curly braces doesn't bring any value addition & non-functional, can we remove it?